src/components/PlainTextReader.vue

```diff
@@ -37,7 +39,7 @@ export default {
 		renderHtml(content) {
 			return '<pre>' + escapeHtml(content) + '</pre>'
 		},
-		extensions: () => [PlainText],
+		extensions: () => [PlainText, CodeBlock],
```

Comment:
Is this change related or shouldn't this be a separate PR?

Reply:
Good point... I'll split!